src/styles/blocks/card.scss

```diff
@@ -0,0 +1,130 @@
+.main {
+  padding: 50px 40px;
+  display: grid;
+  grid-template-columns: repeat($column-count, 200px);
```

Comment:
The variable `$column-count` is used here without being defined at the top of the file or within a scope that includes this line. Ensure that `$column-count` is defined before it is used in the `grid-template-columns` property.